### ISSUE TITLE
Fissure/Invasion/Sortie Notifications, Nightwave and embed links for searchable items from drop rates

### ISSUE THREAD

**Issue by Nickdv9** [WFCD/navis#30]
Hello,
First of all, I would like to thank you for your project, since this is only actively maintained tracker app for warframe right now.

There are three things that I would like to suggest:

**1: Notifications for Fissures/Invasions/Sorties:**
Not sure if this is being worked upon already but I'd like to know if notifications can be expanded to fissures, invasions and sorties or atleast the first two. That way I won't have to rely on discord or twitter notifications for the same.

**2: Nightwave offerings:**
Regarding nightwave, I am curious to know if the offerings can also be displayed along with the daily/weekly challenges like the Baro Inventory. Since the offerings change every week, it would be easier to track them if it were to be integrated in the app itself.

**3: Hyperlinks to warframe wiki:**
The drop rates are quite a handy thing and I really appreciate them being present. However, i think it could be even more helpful to get more information on the aquisition and description of the items. By opening the webpage from wiki as a webview within the app would be a good way to implement this. This could also be done for the Baro inventory as well.

**Comment by TobiTenno** [comment]
For nightwave offerings, DE doesn't really expose that on their data endpoints, so it's be a weekly (or every few weeks?) manual update, which would kinda suck. 

Could integrate the items endpoint to get specific item data for 3, that'd bring the wiki link with it too...

**Comment by SlayerOrnstein** [comment]
I could get started on the invasions, fissures and sortie notifications, actually I think sorties is done just not completed implemented in the app.

**Comment by Nickdv9** [comment]
I see, thank you @TobiTenno @SlayerOrnstein for looking into this

**Comment by TobiTenno** [comment]
I just work on the API, orn does the hard stuff for getting it on navis. Ty for the request!

**Comment by Nickdv9** [comment]
@TobiTenno @SlayerOrnstein This is shaping out really well. Thank you for all your work again. I have used all the betas over the last month and all my suggestions are very well addressed! The feature that allows to filter notifications based on rewards for invasions is really great!
I understand that some features i mentioned are still in development, however there are a couple things that I want to talk about

#1: Codex tweaks:
The new drop down to switch from Drops to Item description is the exact functionality I wanted. However there are certain tweaks that I would like to suggest here. 
When I search for a particular item, I think it would be more seamless if the search term be preserved when I switch between Item descriptions and Drop rates as I am more likely to check the description of the same item I am searching the drop rate for and viceversa.
So to summarize my suggestion, the switch from Item to Drop/Drop to Item preserves the Search query and automatically shows results for the same after the switch, until I choose to search for something else, which would then preserve that query.

#2: Search Issues:
Sometimes I find that an item I search for under 'Items' won't show me any results, while the search for the same item under 'Drops' will give me proper results. An example would be the term 'plastids'. And sometimes if I search for a relic such a 'Axi M1' in 'Items' it does show me all the 4 variant of that relic, but I click 'see wikia' it says 'No valid link provided by API'. So I'm assuming these are API limitations?

**Comment by SlayerOrnstein** [comment]
Alright that's ready for the next release, fissure notifications will be worked on when I add all the resources and feel like invasion notifications is good before moving on

**Comment by TobiTenno** [comment]
@Nickdv9 individual relics don't really have pages of their own. Theoretically they could get tagged and always get the same wiki page, but that also seems unhelpful.

**Comment by Nickdv9** [comment]
@SlayerOrnstein @TobiTenno Thank you for clarification. With the latest update, the codex search term behaviour works exactly the way i had expected and the notifications are very detailed. Thanks again for implementing all the changes :)

**Comment by SlayerOrnstein** [comment]
And [54175ea](https://github.com/WFCD/navis/commit/54175ea3e3a3a4bae76ebfa42424568606842a61) closes this request
